buku

```diff
@@ -2330,7 +2330,7 @@ class BukuDb:
             DELIM separated string of tags.
         """
 
-        tags = tagstr.split(',')
+        tags = (tagstr or '').split(',')
         if not len(tags):
             return tagstr
```

Comment:
This check will never pass.

It needs to be changed to `if not tagstr:`… and moved before the `.split()`.

…Also the `or ''` thing will no longer be necessary after that (though it can be moved into `return`) :sweat_smile: